### DISCUSSION TITLE
Use of `aria-selected` in CSS

### DISCUSSION THREAD

**Discussion by harry-gocity** [ariakit/ariakit#3673] [Questions] [answered]
In the tab example, example styles us `aria-selected`: https://ariakit.org/components/tab

![image](https://github.com/ariakit/ariakit/assets/111427669/7ce1808d-ff36-49e4-ab2b-394220d96d63)

However it's not listed in https://ariakit.org/guide/styling#css-selectors.

I assume so, but is it safe to use?

**Comment by diegohaz** [comment, marked as answer]
Yes, it's safe to use for tabs. It's not mentioned in the styling docs because this attribute is not safe to use as a CSS selector in any other composite item, and that could cause confusion.

In the future, we may introduce a specific `data-selected` attribute that will be consistent across all composite items, but `aria-selected` will still work on tabs.
